x-pack/test/security_solution_cypress/cypress/support/machine_learning.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ML_GROUP_ID } from '@kbn/security-solution-plugin/common/constants';
+import { rootRequest } from '../tasks/api_calls/common';
+
+/**
+ *
+ * Calls the internal ML Module API to set up a module, which installs the jobs
+ * contained in that module.
+ * @param moduleName the name of the ML module to set up
+ * @returns the response from the setup module request
+ */
+export const executeSetupModuleRequest = ({ moduleName }: { moduleName: string }) =>
```

Comment:
not seeing this used in this PR? You mentioned you cherry picked some things so maybe this is just unused but figured I'd bring this up.